src/cclib/parser/gamessukparser.py

```diff
@@ -335,10 +335,14 @@ def extract(self, inputfile, line):
             scfvalues = []
             line = next(inputfile)
             while line.strip():
+                # e.g. **** recalulation of fock matrix on iteration  4 (examples/chap12/pyridine.out)
```

Comment:
It looks like I moved the comment before the line it's referencing. I don't know what the bit in parentheses at the end is, though.